### PR TITLE
Windbg code cleaning

### PR DESCRIPTION
**Pull request by xarkes**

### CODE REVIEW

File: libr/debug/p/debug_windbg.c [hunk 1]
```diff
@@ -117,29 +112,29 @@ static int r_debug_windbg_attach (RDebug *dbg, int pid) {
...
 	return true;
 }
 
 static char *r_debug_windbg_reg_profile(RDebug *dbg) {
 	if (!dbg) return NULL;
-	if (dbg->arch && strcmp (dbg->arch, "x86"))
-		return NULL;
+	if (dbg->arch && strcmp (dbg->arch, "x86")) return NULL;
 	if (dbg->bits == R_SYS_BITS_32) {
 #include "native/reg/windows-x86.h"
```
Comment (xarkes) [inline libr/debug/p/debug_windbg.c]: Does this work actually ? lol Anyway yeah I don't think 32bits is well supported for now.

Comment (radare) [inline libr/debug/p/debug_windbg.c]: yes, its an include directive from the c preprocessor. thats common and standard

Comment (xarkes) [inline libr/debug/p/debug_windbg.c]: I'm talking about the fact that is it included only if the condition is true.

Comment (Maijin) [inline libr/debug/p/debug_windbg.c]: lol

**Pull request merged**